indigo_app/notifications.py

```diff
@@ -51,3 +68,37 @@ def post_save_task(sender, instance, **kwargs):
     if kwargs['created']:
         if isinstance(instance.action_object, Task):
             notifier.send_task_notifications(instance)
+
+
+@receiver(comment_was_posted, sender=Comment)
+def post_comment_save_notification(sender, **kwargs):
+    """
+    Send email when a user comments on a task
+    """
+    task_content_type = ContentType.objects.get_for_model(Task)
+
+    notifier = Notifier()
+
+    comment = kwargs['comment']
```

Comment:
The idea is the Notifier class is the home for this logic -- it makes it easier to test and re-use. So this signal handler just calls `notifier.notify_comment_posted(comment)` and all of this happens in the notifier.